docs/hyn/5.4/spatie-laravel-permission.md

```diff
@@ -143,11 +145,10 @@ $permission = Permission::create(['name' => 'edit articles']);
 $user = User::find(1);
 $user->getRoleNames();
 
-// Assign writer role and save the $user instance to DB.
+// Assign writer role
 $user->assignRole('writer');
-$user->save();
```

Comment:
(Note: `assignRole()` also persists the change, so no need to call `save()`, hence the removal of this line here.)

Reply:
Great to know, thanks.